CHANGELOG.next.asciidoc

```diff
@@ -168,7 +168,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add option to configure docker input with paths {pull}10687[10687]
 - Add Netflow module to enrich flow events with geoip data. {pull}10877[10877]
 - Set `event.category: network_traffic` for Suricata. {pull}10882[10882]
-- Add configuration knob for auto-discover hints to control whether log harvesting is enabled for the pod/container. {issue}10811[10811] {pull}10911[10911]
+- Refactor autodiscover default input settings handling. {pull}12193[12193]
```

Comment:
Should we explictly mention here that it can be used to disable harvesting by default, and to configure CRI paths? I think both cases would worth mention.

Reply:
pushed a more descriptive one, LMKWYT

Reply:
Great, thanks, :shipit: